### ISSUE TITLE
Search Bar Queries in Microsoft Edge are Blocked

### ISSUE THREAD

**Issue by stephendt** [MarcGuiselin/chrometana-pro#15]
Firstly, yes I know I am using Edge instead of Chrome, but I have been using this for over a year with no issues. Unfortunately, as of yesterday I am having a few issues running this extension in Microsoft Edge. When I try to do a search from the search bar, I will get a message that says:

```
lllggmgeiphnciplalhefnbpddbadfdi is blocked

This page has been blocked by Microsoft Edge
ERR_BLOCKED_BY_CLIENT
```

However, if I use the address bar, the search works fine. Not sure if this is an issue with the extension or Edge, but it certainly is annoying.

**Comment by HumanGamer** [comment]
I'm having this issue too. It also asked for a new permission to block web pages recently, not sure if that's related.

**Comment by HumanGamer** [comment]
So I think the issue is the recent changes to the extension. How can I use an older version? The extension worked fine until now.

**Comment by HumanGamer** [comment]
I manually built d304d88632540d31153e3a76a953d8be58ec6871 and installed it as an unpacked extension and it worked.

Steps:
1. install node
2. install grunt
3. clone the repo and in the repo folder, run ``npm install`` followed by ``grunt build-chrome``
4. the unpacked extension is now in the dest folder
5. copy the dest folder somewhere permanent (i put it in D:\Extensions\Chrometana Pro (old)
6. set edge to developer mode and load an unpacked extension passing the folder you copied.

**Comment by MarcGuiselin** [comment]
> It also asked for a new permission to block web pages recently, not sure if that's related.

I switched to the `declarativeNetRequest` extension api since google is phasing out support for v2 extensions and the `webRequest` api. Unfortunately this change required an update to permissions even though "blocking" is never used.

The behavior of this api should be the exact same between chrome and edge. It shouldn't be blocking anything, and redirects correctly in chrome.

I'll investigate and look for a fix tomorrow.

**Comment by MarcGuiselin** [comment]
This issue has been fixed in v 2.2
